README.rst

```diff
@@ -24,17 +27,3 @@ provider) where the strings can be translated.
 
 Compile translated strings by running `make compile_translations`. This will produce a .mo file for each .po in the repo.
 The .mo files are read by Django and are used to provide translations in the running application.
-
-Developing in devstack
-----------------------
-In order to develop in devstack, make sure your devstack is running, then:
-
-#. Switch your devstack to the `edX theme`_
-#. Clone this repo into ``<devstack_folder>/src/``
-#. ``cd <devstack_folder>/devstack/``
-#. ``make dev.shell.credentials``
-#. ``make requirements``
-#. ``pip install -e /edx/src/credentials-themes`` - this will override the version that was installed in the previous step
-#. ``make static``
-
-.. _`edX theme`: https://github.com/openedx/edx-themes/blob/master/docs/guides/local-development.rst
```

Comment:
Why remove this section? I think that we (2U) are the primary consumers of devstack, but I didn't think we were the _only_ consumers. This is still important information to have, is it captured anywhere else?

Reply:
I moved this into the wiki. I believe  someone recently told me we are currently the only consumers of devstack  (I don't know if this is accurate), but more importantly, the instructions are internal now. The dead link for the edx theme is because  it's in [a private repo/(https://github.com/edx/edx-themes/) now.

Reply:
Gotcha, and I may be completely wrong too. I feel better if it exists in our wiki. Thanks for porting the documentation! :)